bigquery_etl/cli/backfill.py

```diff
@@ -364,7 +372,7 @@ def scheduled(ctx, qualified_table_name, sql_dir, project_id, status, json_path=
 @project_id_option(
     ConfigLoader.get("default", "project", fallback="moz-fx-data-shared-prod")
 )
-@billing_project_option()
+@billing_project_option(default=DEFAULT_BILLING_PROJECT)
```

Comment:
Why do we need this as an option again? If there's no billing project on the backfill entry, couldn't we just use the default directly instead of having a CLI option?